README.md

```diff
@@ -19,6 +19,10 @@ hibernate.connection.password=ENC(9ffpQXJi/EPih9o+Xshm5g==)
 
 Specify the encryption key using the `UP_JASYPT_KEY` environment variable.
 
+### [Modal Notification](notification-portlet-webapp/docs/modal.md)
+Modal Notification is a single notice that will be presented until the user clicks the `Accept` button. This feature could be used for a Terms of Service pop-up that must be accepted before
+accessing the portal.
```

Comment:
How about...

    The `modal` display strategy presents notices in a Bootstrap modal dialog.  If the notice 
    defines actions, they will be rendered as buttons at the bottom of the dialog;  any button
    will dismiss the dialog, as well as invoke the spe,cified action server-side.  Notices that
    do not define actions will offer a 'close' (x) option in the upper-right corner of the dialog.  
    This feature could be used for a Terms of Service pop-up that must be accepted before
    accessing the portal.

(Also the same change below.)